### ISSUE TITLE
Fix WARNING: No name was provided for external module "xxx" in output.globals – guessing "xxx"

### ISSUE THREAD

**Issue by flavens** [dasch-swiss/knora-ui-ng-lib#49]
**Issue in terminal when building the library knora-ui**
<img width="1585" alt="Screenshot 2020-03-19 at 10 02 11" src="https://user-images.githubusercontent.com/29532462/77050673-652d4f80-69ca-11ea-8c08-c3728d2b2d16.png">

**Solution**
- add the following lines to `ng-package.json` and `ng-package.prod.json`
```
{
    ...
    "lib": {
       ...,
        "umdModuleIds": {
            "@knora/api": "@knora/api",
            "ngx-color-picker": "ngxColorPicker",
            "jdnconvertiblecalendar": "jdnconvertiblecalendar",
            "jdnconvertiblecalendardateadapter": "jdnconvertiblecalendardateadapter",
            "jdnconvertiblecalendar/dist/src/JDNConvertibleCalendar": "JDNConvertibleCalendar"
        }
    }
}
```

**Comment by tobiasschweizer** [comment]
I already wondered about this warning, great!

**Comment by flavens** [comment]
> I already wondered about this warning, great!

If it is ok with you, I would like to open a PR to fix issue #49 (this one) and #48 together. It's just chore changes. But #48 may bring some breaking changes because all path to knora-ui will have to be updated to @knora-ui

**Comment by flavens** [comment]
For information about the other warnings during the built:
`WARNING: 'ReadValue', 'ReadTextValueAsString', 'ReadIntValue', 'ReadResource', 'ReadBooleanValue', 'ReadDecimalValue', 'ReadColorValue', 'ReadUriValue', 'ReadIntervalValue', 'ReadTimeValue' and 'PropertyDefinition' are imported from external module '@knora/api' but never used`

-> it comes from angular-cli but there is no fix at the moment:
https://github.com/angular/angular/issues/21280

**Comment by tobiasschweizer** [comment]
> If it is ok with you, I would like to open a PR to fix issue #49 (this one) and #48 together. It's just chore changes. But #48 may bring some breaking changes because all path to knora-ui will have to be updated to @knora-ui. Of course I will update as many files as I can in the PR.

Could we wait with #48 until all value components have been implemented?

**Comment by flavens** [comment]
> Could we wait with #48 until all value components have been implemented?

yes, I can wait this both issues because fixing the warnings is not an emergency...
